src/components/auth/EmailConfirmationSent.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { ReactComponent as EmailSent } from '../assets/svgs/mail_sent.svg';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['EmailSent' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5668611)